src/rpc/rpcclient.rs

```diff
@@ -47,7 +47,7 @@ pub trait RpcClient: Send + Sync {
     async fn invoke_method(
         &self,
         method: UUri,
-        request: UPayload,
+        request: UMessage,
```

Comment:
if we accept a UMessage as parameter, then there is no need for priority, ttl and token anymore because these will be contained in UMessage::attributes.